src/components/Menu/MenuItem.tsx

```diff
@@ -55,6 +55,11 @@ type Props<C extends MenuItemElement> = {
    */
   // This prop is required because it's rare to not want a selection handler
   onSelect: ((e: Event) => void) | null;
+  /**
+   * Event callback for when the item is clicked.
+   * @param e
```

Comment:
nit, but this line doesn't seem to be adding much - unless it's necessary for tsdoc to do the right thing, it seems a bit unnecessary.